.meepctl-repocfg.yaml

```diff
@@ -316,3 +318,7 @@ repo:
       meep-sandbox-ctrl-client:
         src: js-packages/meep-sandbox-ctrl-client
         lint: false
+      meep-metrics-engine-client:
```

Comment:
Minor comment: packages are listed in alphabetical order.

Reply:
reordered